packages/fx-core/src/plugins/solution/fx-solution/debug/util/tasksNext.ts

```diff
@@ -341,7 +341,7 @@ function startFuncHostedBot(
     options: {
       cwd: "${workspaceFolder}/bot",
       env: {
-        PATH: "${env:PATH}${command:fx-extension.get-func-path}",
+        PATH: "${command:fx-extension.get-func-path}${env:PATH}",
```

Comment:
Also need to change `fx-solution/debug/util/tasks.ts` to handle existing project add capability?

Reply:
`fx-solution/debug/util/tasks.ts` is not used and it does use `get-func-path`. 

![image](https://user-images.githubusercontent.com/12523371/169635859-eb051ba0-d4a1-4b02-9d83-aa4a07abc639.png)